packages/components/src/Tabs/Tabs.tsx

```diff
@@ -12,10 +12,25 @@ import { Typography } from "../Typography";
 
 interface TabsProps {
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * Specifies the index of the tab that should be active on mount
+   *
+   * @default 0
+   */
+  readonly defaultTab?: number;
+
+  /**
+   * Callback that fires when the active tab changes
+   * @param newTabIndex
+   */
+  onTabChange?(newTabIndex: number): void;
 }
 
-export function Tabs({ children }: TabsProps) {
-  const [activeTab, setActiveTab] = useState(0);
+export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
+  const activeTabInitialValue =
+    defaultTab < React.Children.count(children) ? defaultTab : 0;
```

Comment:
Thinking about this more. I wonder if it's better if it goes to the last index rather than reverting back to 0 when the value is higher than the children's length. 

Feels more forgiving when it goes 

> Oh you want 6 but there's only 5 so maybe you want 5

instead of

> Oh you want 6 but there's only 5 so here's 0

what do you think?

Reply:
Kind of trying to catch a potential off by one error? 

I wonder if this treads to close to "magic". That's why I fallback on the first tab. Kind of like "you told me one that doesn't exist so I'll just be dumb and revert to non-controlled behaviour"

Reply:
hmm.. that's a good point